components/wordPrompt.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  useEffect(() => {
+    fetchWords();
+  }, []);
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.length <= 3) {
+        fetchWords();
+      }
+      lastWords.shift();
+      return [...lastWords];
+    });
+  };
+  const [words, setWords] = useState([]);
+  const fetchWords = () => {
+    fetch("https://random-word-api.herokuapp.com/word?number=10", {
+      method: "GET",
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setWords((lastWords) => {
+          lastWords.push(...data);
+          return [...lastWords];
```

Comment:
you might be able to do `[...lastWords, ...data]` here, then you don't need to do `lastWords.push(...data)`